samples/stock/plotoptions/series-datagrouping-simple-approximation/demo.js

```diff
@@ -0,0 +1,41 @@
+Highcharts.stockChart('container', {
+    rangeSelector: {
+        selected: 4
+    },
+    plotOptions: {
+        series: {
+            tooltip: {
+                valueDecimals: 2
+            },
+            dataGrouping: {
+                forced: true,
+                units: [
+                    ['week', [1]]
+                ]
+            }
+        }
+    },
+    series: [{
+        name: 'ADBE default (average)',
+        data: ADBE
+    }, {
+        name: 'ADBE start',
+        data: ADBE,
+        dataGrouping: {
+            approximation: function (groupData) {
+                return groupData[0];
+            }
+        }
+    }, {
+        name: 'ADBE end',
+        data: ADBE,
+        dataGrouping: {
+            approximation: function (groupData) {
+                var groupEnd = this.dataGroupInfo.start +
+                     this.dataGroupInfo.length - 1;
```

Comment:
This doesn't work. `groupData` contains a list of values that should be used to calculate new, grouped value. For example: `[1, 2, 3]`, which means, DataGrouping wants to group these three points into one. Average would be 2, start=1, end=3.

`this.dataGroupInfo` is used when referring original dataset, not `groupData`.

Simple `return groupData[groupData.length - 1];` should be enough :)